zipkin/src/main/java/com/linecorp/armeria/client/tracing/HttpTracingClient.java

```diff
@@ -137,22 +134,7 @@ private void setRemoteEndpoint(Span span, RequestLog log) {
             address = null;
             port = 0;
         }
-
-        final String remoteServiceName;
-        if (this.remoteServiceName != null) {
-            remoteServiceName = this.remoteServiceName;
-        } else {
-            final String authority = log.requestHeaders().authority();
```

Comment:
What do you think of still following back to authority?

Reply:
Yes, authority should be OK in this case. What do you think, @tacigar ?

Reply:
Ummm...
I'm not sure but I think host name (& port) should not be used for service name too...
@adriancole What do you think?

Reply:
this conversation has happened in the past. there's basically two ways about it.

* some feel they have clean hosts (probably port part of authority is still dubious)
* some have not clean hosts

usually in zipkin don't default service names to dns values as life usually isn't clean. However, this is inconvenient to places that have clean hosts (like cluster scoped, not instance scoped).

this boils down to what causes the least damage vs highest convenience. I personally prefer to keep defaults towards causing least damage as it is harder to clean up later vs ask someone to opt-in.

Reply:
Thank you for your valuable comments 🙇